test/hf_test.py

```diff
@@ -369,6 +392,9 @@ def get_output_loss_value_spec_for_model(model_cls):
                      PLBartForSequenceClassification, PegasusForConditionalGeneration, BartForConditionalGeneration]:
         return {'loss': True, 'logits': False, 'encoder_last_hidden_state': False}
 
+    if model_cls in [ViTForMaskedImageModeling, ViTForImageClassification, ViTModel]:
+        return {'loss': True, 'logits': True}
```

Comment:
Why 'logits': True? True should be specified for the variable that is supposed to be "loss"
```suggestion
        return {'loss': True, 'logits': False}
```

Reply:
sorry that's a holdover from before i figured out that the right API to get the models to return loss at all. thanks for the catch